cmd/self_update_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 OWASP Core Rule Set Project
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+)
+
+type selfUpdateTestSuite struct {
+	suite.Suite
+}
+
+func (s *selfUpdateTestSuite) SetupTest() {
+	rebuildSelfUpdateCommand()
+}
+
+func (s *selfUpdateTestSuite) TearDownTest() {
+}
+
+func TestRunSelfUpdateTestSuite(t *testing.T) {
+	suite.Run(t, new(selfUpdateTestSuite))
+}
```

Comment:
Hm, no tests yet. Is this what you meant by requiring at least two versions?

Reply:
Well, kind of.

I need to find if we can mock things efficiently, but this might imply builds and other stuff (goreleaser, etc).

And the next version we tag will be the first with this feature. Therefore when there is a subsequent version this can be tested properly.

E.g., we release 1.3.9. Only after we release 1.3.10 we can test the update from 9 -> 10.

Reply:
Well, I think now tests look way better.